docs/source/ui/pages/explorers.md

```diff
@@ -161,30 +144,35 @@ By default, metrics aligned by [Step](#step).
 
 By setting metrics alignment to Step, x-axis values will represent the steps of tracked metrics.
 
-<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment_step.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_step" />
+<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment-step.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_step" />
 
 ###### Epoch
 
 By setting metrics alignment to Epoch, x-axis values will represent the epochs of tracked metrics.
 
-<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment_epoch.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_epoch" />
+<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment-epoch.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_epoch" />
 
 ###### Relative Time
 
 By setting metrics alignment to Relative Time, x-axis values will represent by `HH:mm:ss`, duration of tracking process.
 
-<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment_relative.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_relative" />
+<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment-relative.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_relative" />
 
 ###### Absolute Time
 
 By setting metrics alignment to Absolute Time, x-axis values will represent by date `HH:mm:ss D MMM, YY`, since the start date of the first run until the last run.
 
-<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment_absolute.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_absolute" />
+<img src="https://docs-blobs.s3.us-east-2.amazonaws.com/images/ui/pages/metrics-explorer/alignment-absolute.png" style="border-radius: 8px; border: 1px solid #E8F1FC" alt="alignment_absolute" />
 
 ###### Custom Metric
 
 By setting metrics alignment to Custom Metric, x-axis values will represent selected metric values, you can detect correlations between queried metrics and selected metric.
 
+##### Set axes range:
+
+Passing `min` and `max` values for the corresponding axis, will define the axis scale range minimum and maximum bounds (as `[min, max]`) for the chart(s).
```

Comment:
I would go with smth like this `To fix an axis range across all the charts, set the corresponding axis minimum and maximum bounds in the form.`